src/applications/ezr/utils/validation.js

```diff
@@ -51,3 +55,75 @@ export function validatePolicyNumberGroupCode(errors, fieldData) {
     );
   }
 }
+
+export function validateGulfWarDates(
+  errors,
+  { gulfWarStartDate, gulfWarEndDate },
+) {
+  const fromDate = convertToDateField(gulfWarStartDate);
+  const toDate = convertToDateField(gulfWarEndDate);
+  const messages = {
+    range: 'Service end date must be after the service start date',
+    format: 'Enter a date that includes a month and year',
+  };
+
+  if (!isValidDateRange(fromDate, toDate)) {
+    errors.gulfWarEndDate.addError(messages.range);
+  }
+
+  if (fromDate.month.value && !fromDate.year.value) {
+    errors.gulfWarStartDate.addError(messages.format);
+  }
+
+  if (toDate.month.value && !toDate.year.value) {
+    errors.gulfWarEndDate.addError(messages.format);
+  }
+}
+
+export function validateAgentOrangeExposureDates(
+  errors,
+  { agentOrangeStartDate, agentOrangeEndDate },
+) {
+  const fromDate = convertToDateField(agentOrangeStartDate);
+  const toDate = convertToDateField(agentOrangeEndDate);
+  const messages = {
+    range: 'Exposure end date must be after the exposure start date',
+    format: 'Enter a date that includes a month and year',
+  };
+
+  if (!isValidDateRange(fromDate, toDate)) {
+    errors.agentOrangeEndDate.addError(messages.range);
+  }
+
+  if (fromDate.month.value && !fromDate.year.value) {
+    errors.agentOrangeStartDate.addError(messages.format);
+  }
+
+  if (toDate.month && !toDate.year) {
```

Comment:
Add `.value` to each of these, as are done above. This is a mistake I made when creating this function.

Reply:
Thank for that catch Matt. I have addressed this in my recent changes.